client/my-sites/checkout/cart/cart-messages.tsx

```diff
@@ -97,10 +97,9 @@ function getBlockedPurchaseErrorMessage( {
 			components: {
 				a: (
 					<a
-						href={
-							'https://wordpress.com/error-report/' +
-							( selectedSiteSlug ? '?url=payment@' + selectedSiteSlug : '' )
-						}
+						href={ `https://wordpress.com/account-assistance-form/${
+							selectedSiteSlug ? '?url=payment@' + selectedSiteSlug : ''
+						}` }
```

Comment:
I actually prefer the syntax used before. Having a ternary that returns a string inside a string template is kind of strange to read.